### ISSUE TITLE
`dir_info` has a different type for size than `fs_bytes`

### ISSUE THREAD

**Issue by Moohan** [r-lib/fs#323]
As title, if you run `dir_info()`, the size variable has a `class() = "fs_bytes"` however if you use `fs_bytes()` it has class `"fs_bytes" "numeric"`.

I have code that used to work:
```r
mutate(size = case_when(
      size < 0 | is.na(size) ~ fs_bytes(0),
      TRUE ~ size
    ))

Error: must have class `fs_bytes/numeric`, not class `fs_bytes`
```
I think this is due to an update with `dplyr` where the type checking is now more strict but I now get an error.

**Comment by gaborcsardi** [comment]
I have this now with fs 1.5.1, this is now solved?
```r
> class(dir_info()$size)
[1] "fs_bytes" "numeric"
```

**Comment by Moohan** [comment]
Yep seems to be working now :) I would guess it was fixed in 1.5.0 ?
